returnn/tf/layers/basic.py

```diff
@@ -4769,39 +4773,52 @@ def __init__(self, eval, **kwargs):
 
 class CompareLayer(LayerBase):
   """
-  This layer performs an elementwise comparison of all provided sources, and returnns a boolean vector
-  of the same shape.
-  If the `value` parameter is provided, all elements in the provided sources are also compared to this value.
+  Compares element-wise the tokens of all input sequences among themselves and/or with a specified given value.
+  The comparisons are performed in a chain according to the order in which they are listed.
+
+  Example::
+
+    {"class": "compare", "from": ["i1", "i2"], "value": val, "kind": "less"}
```

Comment:
Also newline after the code example.

Reply:
I think the code example needs to be indented by 4 spaces (after the `::`) (right @JackTemaki ?).

Reply:
Should I use tab or spaces for indenting?

Reply:
Always spaces.

Reply:
Please follow the PyCharm settings as described in the wiki.

Reply:
> I think the code example needs to be indented by 4 spaces (after the `::`) (right @JackTemaki ?).

I think the depth of the indentation is not relevant, I checked the build and it compiles as it is now.

Reply:
Maybe because it is already 4 spaces from the very left? Maybe it was also for PyCharm (which also parses the docstring and formats it). I don't remember anymore. Or maybe I just remember wrong. But shouldn't really matter, as long as it is indented.